pylib/anki/notes.py

```diff
@@ -32,6 +32,8 @@ class Note(DeprecatedNamesMixin):
     data = ""
     id: NoteId
     mid: NotetypeId
+    # used when creating a copy of an existing note
+    orig_note_id: NoteId | None = None
```

Comment:
Adding a property here for an editor-specific feature feels a bit unclean. What about storing it in the editor instance instead?